proto/spawn_diff.proto

```diff
@@ -66,17 +65,19 @@ message FileSetDiff {
 }
 
 message FileDiff {
+  // The shared logical path of the diffed files, before resolving any symlinks.
+  string logical_path = 1;
   oneof old {
-    tools.protos.ExecLogEntry.File old_file = 1;
-    tools.protos.ExecLogEntry.Directory old_directory = 3;
-    tools.protos.ExecLogEntry.UnresolvedSymlink old_symlink = 5;
-    string old_invalid_output = 7;
+    tools.protos.ExecLogEntry.File old_file = 2;
+    tools.protos.ExecLogEntry.Directory old_directory = 4;
+    tools.protos.ExecLogEntry.UnresolvedSymlink old_symlink = 6;
+    string old_invalid_output = 8;
   }
   oneof new {
-    tools.protos.ExecLogEntry.File new_file = 2;
-    tools.protos.ExecLogEntry.Directory new_directory = 4;
-    tools.protos.ExecLogEntry.UnresolvedSymlink new_symlink = 6;
-    string new_invalid_output = 8;
+    tools.protos.ExecLogEntry.File new_file = 3;
+    tools.protos.ExecLogEntry.Directory new_directory = 5;
+    tools.protos.ExecLogEntry.UnresolvedSymlink new_symlink = 7;
+    string new_invalid_output = 9;
```

Comment:
Is this a safe change? Not sure if we should add it as tag 9 instead of tag 1

Reply:
See above, this proto is internal only so far.